src/kpro_connection.erl

```diff
@@ -314,8 +323,11 @@ call(Pid, Request) ->
       {error, {connection_down, Reason}}
   end.
 
-reply({To, Tag}, Reply) ->
-  To ! {Tag, Reply}.
+maybe_reply({To, Ref}, Reply) when is_reference(Ref) ->
```

Comment:
Would be great to have a spec here since logic depends in the type of ```Ref```

Reply:
done